apps/ai-image-generator/frontend/package.json

```diff
@@ -7,6 +7,7 @@
     "@contentful/f36-components": "4.45.0",
     "@contentful/f36-tokens": "4.0.4",
     "@contentful/integration-component-library": "^1.1.41",
+    "@contentful/integration-frontend-toolkit": "file:~/contentful-integration-frontend-toolkit-1.0.0-semantic-release.tgz",
```

Comment:
This will obviously not be the correct import, it needs to be updated with the correct version of the integration-frontend-toolkit once that is versioned and released correctly.